### PR TITLE
1438 browserstack some python tests fails on browserstack but pass on our real devices

### PR DESCRIPTION
**Pull request by RobertPoienar**

### CODE REVIEW

**Comment by RobertPoienar** [comment]
@alttester/developers

File: Assets/Editor/BuildAltTester.cs [hunk 1]
```diff
@@ -33,230 +34,70 @@ protected static void MacBuildFromCommandLine()
...
-                EditorApplication.Exit(1);
-#endif
+                var buildPlayerOptions = GetBuildPlayerOptions("sampleGame", BuildTarget.StandaloneOSX);
+                buildGame(buildPlayerOptions, BuildTargetGroup.Standalone);
 
             }
             catch (Exception exception)
             {
                 logger.Error(exception);
-                // EditorApplication.Exit(1);
```
Comment (gomboskriszta15) [inline Assets/Editor/BuildAltTester.cs:98]: Do we still need this?

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -86,7 +86,7 @@ jobs:
     runs-on: [self-hosted, macOS]
     needs: paths-filter
 
-    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development' || needs.paths-filter.outputs.proxy == 'true' || needs.paths-filter.outputs.scenes == 'true' || needs.paths-filter.outputs.python == 'true'
+    #  if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/development' || needs.paths-filter.outputs.proxy == 'true' || needs.paths-filter.outputs.scenes == 'true' || needs.paths-filter.outputs.python == 'true'
```
Comment (rucindrea) [inline .github/workflows/push.yml:89]: Do we need to uncomment these?

File: Assets/Editor/BuildAltTester.cs [hunk 3]
```diff
@@ -355,8 +148,93 @@ private static AltInstrumentationSettings getInstrumentationSettings()
                 instrumentationSettings.AltServerPort = 13010;
             }
             instrumentationSettings.ResetConnectionData = true;
+            instrumentationSettings.UID = UnityEngine.SystemInfo.deviceUniqueIdentifier.ToString() + DateTimeOffset.Now.ToUnixTimeSeconds().ToString();
 
             return instrumentationSettings;
         }
+        public static string[] GetScene()
```
Comment (rucindrea) [inline Assets/Editor/BuildAltTester.cs:155]: This should probably be GetScenes?

**Review by gomboskriszta15 — APPROVED** [review]

**Pull request merged**